pygfx/renderers/wgpu/engine/renderer.py

```diff
@@ -440,21 +455,43 @@ def visit_wobject(ob):
                     ambient_color[0] += r * ob.intensity
                     ambient_color[1] += g * ob.intensity
                     ambient_color[2] += b * ob.intensity
-
-        # Flatten the scenegraph, categorised by render_order
-        wobject_dict = {}
+            else:
+                # Add to semi-flat data structure
+                # wobject_dict.setdefault(ob.render_order, []).append(ob)
+                if ob.material:
+                    from ....helpers import Stats
+
+                    if isinstance(ob.parent, Stats):  # special case for Stats
+                        flat.wobjects["front"].append(ob)
+                    elif getattr(ob.material, "transmission", None):
```

Comment:
```suggestion
elif hasattr(ob.material, "transmission"):
```

Reply:
Here, "transmission" may be 0, and when it is 0, it is treated as non-transmissive object.

Reply:
Ah! That's not immediately obvious from the code. I get it.